homeassistant/components/miele/const.py

```diff
@@ -325,13 +327,17 @@ class MieleActions(IntEnum):
     MieleAppliance.ROBOT_VACUUM_CLEANER: STATE_PROGRAM_PHASE_ROBOT_VACUUM_CLEANER,
 }
 
-STATE_PROGRAM_TYPE = {
-    0: "normal_operation_mode",
-    1: "own_program",
-    2: "automatic_program",
-    3: "cleaning_care_program",
-    4: "maintenance_program",
-}
+
+class STATE_PROGRAM_TYPE(MieleEnum):
```

Comment:
Please comply with the styling we use for classes, like:

```suggestion
class StateProgramType(MieleEnum):
```